<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>368</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>368</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>创建进程</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 9pt &quot;Consolas&quot;,&quot;微软雅黑&quot;;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_0">
   <item alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="add_resource">
     <property name="text">
      <string>添加资源</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="resource_list_box">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>资源列表</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_1">
      <item>
       <widget class="QScrollArea" name="scroll_area">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>512</width>
          <height>368</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scroll_area_layout">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>510</width>
           <height>366</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="resource_0" native="true">
            <property name="index" stdset="0">
             <UInt>0</UInt>
            </property>
            <property name="ishide" stdset="0">
             <bool>false</bool>
            </property>
            <layout class="QHBoxLayout" name="resource_0">
             <item>
              <widget class="QLabel" name="resource_label_0">
               <property name="text">
                <string>资源0</string>
               </property>
               <property name="index" stdset="0">
                <UInt>0</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="combo_box_0">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="index" stdset="0">
                <UInt>0</UInt>
               </property>
               <item>
                <property name="text">
                 <string>CPU</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GPU</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>IO</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_0">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <UInt>32</UInt>
               </property>
               <property name="pageStep">
                <UInt>10</UInt>
               </property>
               <property name="value">
                <UInt>0</UInt>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
               <property name="index" stdset="0">
                <UInt>0</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="time_label_0">
               <property name="text">
                <string>0秒</string>
               </property>
               <property name="index" stdset="0">
                <UInt>0</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="remove_resource_0">
               <property name="text">
                <string>删除资源</string>
               </property>
               <property name="index" stdset="0">
                <UInt>0</UInt>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="resource_1" native="true">
            <property name="index" stdset="0">
             <UInt>1</UInt>
            </property>
            <property name="ishide" stdset="0">
             <bool>false</bool>
            </property>
            <layout class="QHBoxLayout" name="resource_1">
             <item>
              <widget class="QLabel" name="resource_label_1">
               <property name="text">
                <string>资源1</string>
               </property>
               <property name="index" stdset="0">
                <UInt>1</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="combo_box_1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="index" stdset="0">
                <UInt>1</UInt>
               </property>
               <item>
                <property name="text">
                 <string>CPU</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GPU</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>IO</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <UInt>32</UInt>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="index" stdset="0">
                <UInt>1</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="time_label_1">
               <property name="text">
                <string>0秒</string>
               </property>
               <property name="index" stdset="0">
                <UInt>1</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="remove_resource_1">
               <property name="text">
                <string>删除资源</string>
               </property>
               <property name="index" stdset="0">
                <UInt>1</UInt>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="resource_2" native="true">
            <property name="index" stdset="0">
             <UInt>2</UInt>
            </property>
            <property name="ishide" stdset="0">
             <bool>false</bool>
            </property>
            <layout class="QHBoxLayout" name="resource_2">
             <item>
              <widget class="QLabel" name="resource_label_2">
               <property name="text">
                <string>资源2</string>
               </property>
               <property name="index" stdset="0">
                <UInt>2</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="combo_box_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="index" stdset="0">
                <UInt>2</UInt>
               </property>
               <item>
                <property name="text">
                 <string>CPU</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GPU</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>IO</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <UInt>32</UInt>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="index" stdset="0">
                <UInt>2</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="time_label_2">
               <property name="text">
                <string>0秒</string>
               </property>
               <property name="index" stdset="0">
                <UInt>2</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="remove_resource_2">
               <property name="text">
                <string>删除资源</string>
               </property>
               <property name="index" stdset="0">
                <UInt>2</UInt>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="resource_3" native="true">
            <property name="index" stdset="0">
             <UInt>3</UInt>
            </property>
            <property name="ishide" stdset="0">
             <bool>false</bool>
            </property>
            <layout class="QHBoxLayout" name="resource_3">
             <item>
              <widget class="QLabel" name="resource_label_3">
               <property name="text">
                <string>资源3</string>
               </property>
               <property name="index" stdset="0">
                <UInt>3</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="combo_box_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="index" stdset="0">
                <UInt>3</UInt>
               </property>
               <item>
                <property name="text">
                 <string>CPU</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GPU</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>IO</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <UInt>32</UInt>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="index" stdset="0">
                <UInt>3</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="time_label_3">
               <property name="text">
                <string>0秒</string>
               </property>
               <property name="index" stdset="0">
                <UInt>3</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="remove_resource_3">
               <property name="text">
                <string>删除资源</string>
               </property>
               <property name="index" stdset="0">
                <UInt>3</UInt>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="resource_4" native="true">
            <property name="index" stdset="0">
             <UInt>4</UInt>
            </property>
            <property name="ishide" stdset="0">
             <bool>false</bool>
            </property>
            <layout class="QHBoxLayout" name="resource_4">
             <item>
              <widget class="QLabel" name="resource_label_4">
               <property name="text">
                <string>资源4</string>
               </property>
               <property name="index" stdset="0">
                <UInt>4</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="combo_box_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="index" stdset="0">
                <UInt>4</UInt>
               </property>
               <item>
                <property name="text">
                 <string>CPU</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GPU</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>IO</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <UInt>32</UInt>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="index" stdset="0">
                <UInt>4</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="time_label_4">
               <property name="text">
                <string>0秒</string>
               </property>
               <property name="index" stdset="0">
                <UInt>4</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="remove_resource_4">
               <property name="text">
                <string>删除资源</string>
               </property>
               <property name="index" stdset="0">
                <UInt>4</UInt>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="resource_5" native="true">
            <property name="index" stdset="0">
             <UInt>5</UInt>
            </property>
            <property name="ishide" stdset="0">
             <bool>false</bool>
            </property>
            <layout class="QHBoxLayout" name="resource_5">
             <item>
              <widget class="QLabel" name="resource_label_5">
               <property name="text">
                <string>资源5</string>
               </property>
               <property name="index" stdset="0">
                <UInt>5</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="combo_box_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="index" stdset="0">
                <UInt>5</UInt>
               </property>
               <item>
                <property name="text">
                 <string>CPU</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GPU</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>IO</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <UInt>32</UInt>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="index" stdset="0">
                <UInt>5</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="time_label_5">
               <property name="text">
                <string>0秒</string>
               </property>
               <property name="index" stdset="0">
                <UInt>5</UInt>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="remove_resource_5">
               <property name="text">
                <string>删除资源</string>
               </property>
               <property name="index" stdset="0">
                <UInt>5</UInt>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="create_thread">
     <property name="text">
      <string>创建进程</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
